wordpress-mobile/WordPress-iOS
# [Domain Management] Choice Screen UI Updates

Fixes #21978 

# Testing Steps
1. Setup the store sandbox
2. Navigate to the "Me" tab
3. Tap on "My Domains"
4. Tap on the + button
5. Search for a domain
6. Select a .live domain
7. Tap on "Select Domain"
8. ✅ Make sure the purchase choice screen UI matched the designs

# Screenshot Reference
| Light Theme  | Dark Theme |
| ------------- | ------------- |
| ![Simulator Screenshot - iPhone 15 Pro - 2023-11-08 at 13 01 45](https://github.com/wordpress-mobile/WordPress-iOS/assets/15968946/11d5ab28-5821-425e-a670-23c83d08ff44) | ![Simulator Screenshot - iPhone 15 Pro - 2023-11-08 at 13 01 40](https://github.com/wordpress-mobile/WordPress-iOS/assets/15968946/d8d9d210-7ea3-46f1-a661-79b9b2572a00)|

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:


UI Changes testing checklist:
